MovieDatabase/MovieDatabase/Dto/MovieDto.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace MovieDatabase.Dto
+{
+    public class MovieDto
+    {
+        public int Id { get; set; }
```

Comment:
add director name string here and update the mapper profile to map the director name to it